### DISCUSSION TITLE
Math Track

### DISCUSSION THREAD

**Discussion by rutefig** [rutefig/zklings#1] [Q&A]
What topics, questions or ideas do you think should be beneficial to add for the math exercises?

**Comment by Dozie2001** [comment]
A prerequisite course probably touching maybe polynomials and matrices

**Reply by Pfed-prog** [reply]
https://online.stat.psu.edu/statprogram/reviews/calculus/summations-and-series

**Reply by Pfed-prog** [reply]
division by 0

**Comment by chee-chyuan** [comment]
introduction to groups and fields maybe?

**Reply by Pfed-prog** [reply]
https://stackoverflow.com/a/78599231/13943679

https://mathematics.science.narkive.com/omnKNUuZ/in-mathematics-what-do-z-pz-and-z-nz-stand-for

**Reply by Pfed-prog** [reply]
https://www.khanacademy.org/computing/computer-science/cryptography/modarithmetic/a/what-is-modular-arithmetic

**Reply by Pfed-prog** [reply]
https://www.britannica.com/science/modular-arithmetic

**Reply by Pfed-prog** [reply]
for fields https://www.britannica.com/science/Fermats-theorem
